static/docs/commands-reference/config.md

```diff
@@ -87,74 +87,45 @@ files](/doc/user-guide/dvc-files-and-directories) for more details.)
   The default value is `cache`, which resolved relative to the default project
   config location results in `.dvc/cache`.
   > See also helper command `dvc cache dir` that properly transform paths
-  relative to the present working directory into relative to the project config
-  file.
+  > relative to the present working directory into relative to the project
+  > config file.
 
 - `cache.protected` - makes files in the workspace read-only. Possible values
   are `true` or `false` (default). Run `dvc checkout` for the change go into
-  effect. (It affects only files that are under DVC control.)
+  effect. (It affects only files that are under DVC control.)  
```

Comment:
remove trailing whitespaces

Reply:
They are in on purpose. In markdown they help to insert a line break within a paragraph.

Explained here: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists

Reply:
So, is this an actual whitespace? I'm fine with that but I would expect it to be fragile - some editors remove trailing whitespaces on save and prettier won't return them most likely. Not a big deal though, let's not worry about this now.

Reply:
You're right. We may want to avoid these in the future... I'll search all the docs to see if there's many occurrences.